cookbooks/aws-parallelcluster-slurm/recipes/config/config_compute.rb

```diff
@@ -24,7 +24,7 @@
   mode '0700'
 end
 
-include_recipe 'aws-parallelcluster-slurm::mount_slurm_dir'
+include_recipe 'aws-parallelcluster-slurm::mount_slurm_dir' if node['cluster']['internal_shared_storage_type'] == 'ebs'
```

Comment:
There is something strange here. I mean the code is correct, but seems counterintuitive that we're mounting the slurm dir only when using ebs.

I think it's better to move this if into the `mount_slurm_dir` recipe itself. It's responsibility of the recipe to know if must be executed or not. In this way we'll avoid the same check in the config_login and config_compute.

Reply:
Yup, I can do this.